src/arrayMethodSort.js

```diff
@@ -4,9 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
+        }
+      }
+    }
```

Comment:
Consider optimizing the bubble sort algorithm by adding a flag that checks if any swaps have occurred in the inner loop. If no swaps occur, the array is already sorted, and you can break early to improve performance.